LICENSES/BSD-3-Clause.txt

```diff
@@ -0,0 +1,11 @@
+Copyright (c) <year> <owner>. All rights reserved.
```

Comment:
Is the date and copyright owner supposed to be filled in here?

Reply:
No.